src/org/centos/pipeline/PackagePipelineUtils.groovy

```diff
@@ -91,6 +91,63 @@ def setMessageFields(String messageType, String artifact) {
     return [ 'topic': topic, 'properties': messagePropertiesString, 'content': messageContentString ]
 }
 
+/**
+ * Method that uses contra-lib shared library
+ * to create the ci.artifact.test.messageType messages
+ * @param messageType: queued, running, complete, error
+ * @param artifact: dist-git-pr, koji-build
+ * @return
+ */
+// Plan is to rename to setMessageFields and remove function above once everything seems fine and stable
+def setTestMessageFields(String messageType, String artifact) {
+    // Set values that go in multiple closures
+    myTopic = "${MAIN_TOPIC}.ci.${artifact}.test.${messageType}"
+    myType = (artifact == 'koji-build') ? 'tier0' : 'build'
```

Comment:
@thrix please ack/nack the MESSAGE_TYPES I chose

Reply:
Yeah, I think this is ok for Fedora pipeline. with dist-git-pr artifact we choose build, yep